web/js/components/layer/product-picker/search/product-facet.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Facet } from '@elastic/react-search-ui';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faCaretDown,
+  faCaretRight,
+  faInfoCircle,
+} from '@fortawesome/free-solid-svg-icons';
+import { Tooltip } from 'reactstrap';
+
+function ProductFacet(props) {
+  const [tooltipVisible, toggleTooltip] = useState(false);
+
+  const {
+    config,
+    data,
+    collapsed,
+    toggleCollapse,
+  } = props;
+
+  const {
+    field,
+    booleanOptionLabel,
+    label,
+    filterType,
+    tooltip,
+    show,
+    view,
+  } = config;
+
+  const renderHeaderIcons = () => (
+    <>
+      <Tooltip
+        className="facet-tooltip-content"
+        isOpen={tooltipVisible}
+        target={`${field}-tooltip-target`}
+        placement="right"
+        toggle={() => toggleTooltip(!tooltipVisible)}
+        delay={{ show: 0, hide: 1000 }}
```

Comment:
I recommend reducing the `hide` time to `300`-ish ms.

Reply:
Good catch, meant to change this up after the tooltip with a link in it was removed!